netboot/netconf_test.go

```diff
@@ -7,41 +7,37 @@ import (
 
 	"github.com/insomniacslk/dhcp/dhcpv4"
 	"github.com/insomniacslk/dhcp/dhcpv6"
-	"github.com/insomniacslk/dhcp/iana"
 	"github.com/stretchr/testify/require"
 )
 
-func getAdv(modifiers ...dhcpv6.Modifier) *dhcpv6.Message {
+func getAdv(solModifiers, advModifiers []dhcpv6.Modifier) *dhcpv6.Message {
```

Comment:
Why does it take two different lists of modifiers? If we are not using it, I'll rather leave it how it was before.

Reply:
some modifiers must be applied only to the advertise - they were applied to the solicit too. With the latest iteration no modifiers are applied to the solicit anymore, so this could go away (until needed again)

Reply:
If it is not used, I would remove it :P

Reply:
it's removed in the last push